domains/bens.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "My portfolio",
+  "repo": "https://github.com/coco-droid",
+  "owner": {
+    "username": "coco-droid",
+    "email": "robensjeanbaptiste29@gmail.com"
+  },
+  "record": {
+    "URL": "https://bit.ly/bensdev"
```

why redirect to a redirect website?
```suggestion
    "URL": "https://robe.alwaysdata.net/"
```